Dockerfile

```diff
@@ -0,0 +1,31 @@
+FROM python:3.9 as requirements-stage
```

Comment:
python:3.9-slim-bullseye is more lightweight image that we could consider– a pretty good discussion on base images can be found here:
https://pythonspeed.com/articles/base-image-python-docker-images/

Reply:
Lightweight is good.  I'm convinced!